libs/repositories/src/UsdRepository/UsdRepositoryCoingecko.ts

```diff
@@ -60,7 +68,7 @@ export class UsdRepositoryCoingecko implements UsdRepository {
       response
     );
 
-    return priceData[tokenAddressLower].usd;
+    return priceData[tokenAddressLower].usd || null;
```

Comment:
After the check above, can this still be falsy?

Reply:
you are right, it shouldn't